flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
this is really unfortunate.

Reply:
Due to some native dependencies I have to stick to x86 unfortunately

Reply:
but it's GPL-3....

Reply:
It's not really a hard breaking dependency but I don't wanna publish for an arch that's not properly tested. 

I am trying my best to get it tested on archs like aarch64 but for now I request that this be released for x86 only